### ISSUE TITLE
Publish the `v0.1.0` under `org.http4s`

### ISSUE THREAD

**Issue by danicheg** [http4s/http4s-grpc#114]
Since #105 is merged, I think we can move on and publish `v0.1.0`. Any objections? cc @http4s/grpc

**Comment by hamnis** [comment]
can we do that without https://github.com/http4s/http4s-grpc/issues/98 ?

**Comment by danicheg** [comment]
Yeah, I don't think we must change the license. If we're doing the license transition properly, we should get the consent of the original authors.

**Comment by danicheg** [comment]
Actually, I somehow forget about this issue https://github.com/http4s/http4s-grpc/issues/70. It seems somewhat crucial to fix before releasing.

**Comment by hamnis** [comment]
I think we should release this as is, then fix #70 . There are number of improvements in the library as is.

**Comment by hamnis** [comment]
@danicheg seems like Chris is not replying to inquiries, so should we try to release this as a M1 without license change?

**Comment by rossabaker** [comment]
There are three plausible ways forward:

1. Get approval from all original contributors who made a significant contribution.  This one's stuck.
2. Just keep it MIT.  It's not consistent with the rest of http4s, but, so?  Both MIT and Apache-2.0 licensed code are on the classpath already because of cats-effect and fs2, and I'll award Two Internet Points to anyone who can guess which is which without looking.
3. MIT source files can be folded into an Apache-2.0 licensed work as long as all original copyright statements are preserved, typically in reverse chronological order.  Last time I had to do similar, I found sbt-header leaves things alone outside the first block comment.

The easy button here is probably option 2, but I'm cool with whatever.

**Comment by vlovgr** [comment]
I went with option 2 in https://github.com/http4s/http4s-grpc/pull/222. Closed #98. Please let me know or create a pull request if something looks incorrect.

I'll go ahead and release v0.1.0 shortly without https://github.com/http4s/http4s-grpc/issues/70, since we already have plenty of unreleased improvements.
